spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/Schema.scala

```diff
@@ -348,4 +348,21 @@ object Schema extends Logging {
         throw new IOException(errorMessage)
       }
     }
+
+  /**
+    * Get default TTL for Table
+    */
+  def ttlFromCassandra(
+    connector: CassandraConnector,
+    keyspaceName: String,
+    tableName: String) : Option[Int] = {
+
+    tableFromCassandra(connector, keyspaceName, tableName) // Checks for existence of table
+    Try(connector.withClusterDo(
```

Comment:
This may be the wrong thing to do, but without atomic metadata updates this can throw a null pointer on very recently created tables ... Maybe we have this sleep and retry?